.devcontainer/devcontainer.json

```
@@ -1,5 +1,5 @@
 {
-	"image": "ghcr.io/juliushaertl/nextcloud-dev-php80:latest",
+	"image": "ghcr.io/juliushaertl/nextcloud-dev-php82:latest",
```

Should be this one these days ;)

```suggestion
	"image": "ghcr.io/juliusknorr/nextcloud-dev-php82:latest",
```